api/src/test/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandlerTest.java

```diff
@@ -0,0 +1,255 @@
+
+    private EncounterService encounterService;
+
+    /**
+     * Runs before every test and sets patientService and encounterService
+     */
+    @Before
+    public void runBeforeEveryTest() {
+        patientService = Context.getPatientService();
+        encounterService = Context.getEncounterService();
+    }
+
+    /**
+     * Creates and returns a Patient
+     * 
+     * @return PPatient object
+     */
+    private Patient createPatient()
+    {
+        Patient patient = new Patient();
+        
+        PersonName pName = new PersonName();
+        pName.setGivenName("Tom");
+        pName.setMiddleName("E.");
+        pName.setFamilyName("Patient");
+        patient.addName(pName);
+        
+        PersonAddress pAddress = new PersonAddress();
+        pAddress.setAddress1("123 My street");
+        pAddress.setAddress2("Apt 402");
+        pAddress.setCityVillage("Anywhere city");
+        pAddress.setCountry("Some Country");
+        Set<PersonAddress> pAddressList = patient.getAddresses();
+        pAddressList.add(pAddress);
+        patient.setAddresses(pAddressList);
+        patient.addAddress(pAddress);
+        // patient.removeAddress(pAddress);
+        
+        patient.setBirthdateEstimated(true);
+        patient.setBirthdate(new Date());
+        patient.setBirthdateEstimated(true);
+        patient.setDeathDate(new Date());
+        patient.setCauseOfDeath(new Concept());
+        patient.setGender("male");
+        
+        List<PatientIdentifierType> patientIdTypes = patientService.getAllPatientIdentifierTypes();
+        assertNotNull(patientIdTypes);
+        PatientIdentifier patientIdentifier = new PatientIdentifier();
+        patientIdentifier.setIdentifier("123-0");
+        patientIdentifier.setIdentifierType(patientIdTypes.get(0));
+        patientIdentifier.setLocation(new Location(1));
+        patientIdentifier.setPreferred(true);
+        
+        Set<PatientIdentifier> patientIdentifiers = new LinkedHashSet<>();
+        patientIdentifiers.add(patientIdentifier);
+        
+        patient.setIdentifiers(patientIdentifiers);
+        
+        patientService.savePatient(patient);
+
+        return patient;
+    }
+
+    /**
+     * Creates and returns Person, given name
+     * 
+     * @param name name of the person to be created
+     * @return Person object
+     */
+    private Person newPerson(String name) {
+        Person person = new Person();
+        Set<PersonName> personNames = new TreeSet<>();
+        PersonName personName = new PersonName();
+        personName.setFamilyName(name);
+        personNames.add(personName);
+        person.setNames(personNames);
+        person.setPersonDateCreated(new Date());
+        return person;
+    }
+
+    /**
+     * Creates and returns Encounter, given the patient's Uuid
+     * 
+     * @param patient_uuid Uuid of the patient 
+     * @return Encounter object
+     */
+    private Encounter createEncounter(String patient_uuid)
+    {
```

Comment:
Properly format it as `private Encounter createEncounter(String patient_uuid) {`